cmd/svcat/svcat_test.go

```diff
@@ -187,10 +187,10 @@ func TestCommandOutput(t *testing.T) {
 		{name: "list all brokers", cmd: "get brokers", golden: "output/get-brokers.txt"},
 		{name: "list all brokers (json)", cmd: "get brokers -o json", golden: "output/get-brokers.json"},
 		{name: "list all brokers (yaml)", cmd: "get brokers -o yaml", golden: "output/get-brokers.yaml"},
-		{name: "get broker", cmd: "get broker ups-broker", golden: "output/get-broker.txt"},
-		{name: "get broker (json)", cmd: "get broker ups-broker -o json", golden: "output/get-broker.json"},
-		{name: "get broker (yaml)", cmd: "get broker ups-broker -o yaml", golden: "output/get-broker.yaml"},
-		{name: "describe broker", cmd: "describe broker ups-broker", golden: "output/describe-broker.txt"},
+		{name: "get broker (cluster)", cmd: "get broker ups-broker --scope cluster", golden: "output/get-broker.txt"},
+		{name: "get broker (cluster)(json)", cmd: "get broker ups-broker --scope cluster -o json", golden: "output/get-broker.json"},
+		{name: "get broker (cluster)(yaml)", cmd: "get broker ups-broker --scope cluster -o yaml", golden: "output/get-broker.yaml"},
+		{name: "describe broker (cluster)", cmd: "describe broker ups-broker --scope cluster", golden: "output/describe-broker.txt"},
```

Comment:
nit. I don't think the paren notation is appropriate here. should use words "get cluster scoped" or some such.

Reply:
yeah, I think you're right, will fix